### PR TITLE
Fix some lint errors

### PR DESCRIPTION
**Pull request by rjoleary**
Some of these might actually be bugs -- the error code is not being
checked properly.

Signed-off-by: Ryan O'Leary <ryan@ryanoleary.ca>

### CODE REVIEW

File: cmds/ed/command.go [hunk 1]
```diff
@@ -41,7 +42,8 @@ func Command(f Editor, c string, startLine, endLine int) error {
 	case 'r':
 		fname := strings.TrimLeft(a, " \t")
 		debug("read %v @ %v, %v", f, startLine, endLine)
-		r, err := os.Open(fname)
```
Comment (rjoleary) [inline cmds/ed/command.go]: It is a bit subtle. This `:=` defines a new `err` which shadows the original definition on line 16.

File: cmds/ed/command.go [hunk 2]
```diff
@@ -41,7 +42,8 @@ func Command(f Editor, c string, startLine, endLine int) error {
...
+		var r io.Reader
+		r, err = os.Open(fname)
 		debug("%v: r is %v, err %v", fname, r, err)
 		if err == nil {
 			_, err = f.Read(r, startLine, endLine)
```
Comment (rjoleary) [inline cmds/ed/command.go]: So this `err` is never checked because it immediately goes out of scope.

Comment (rminnich) [reply cmds/ed/command.go]: ed is kinda crap. sorry about that.

File: cmds/io/io.go [hunk 3]
```diff
@@ -68,7 +68,8 @@ func main() {
 	case "inb", "inw", "inl":
 		err = in(addr, i.val)
 	case "outb", "outw", "outl":
-		v, err := strconv.ParseUint(a[2], 0, i.valbits)
+		var v uint64
```
Comment (rminnich) [inline cmds/io/io.go]: I apologize for this ...

**Comment by rjoleary** [comment]
@tpepper I agree your point is valid. If we can break down the functions in smaller functions, the code flow will be simpler and this issue will likely evaporate. However, since this is the ed command, it is not the top priority.

**Pull request merged**